<!DOCTYPE html>
<html>
  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <link href='https://fonts.googleapis.com/css?family=Chivo:900' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen" />
    <link rel="stylesheet" type="text/css" href="stylesheets/pygment_trac.css" media="screen" />
    <link rel="stylesheet" type="text/css" href="stylesheets/print.css" media="print" />
    <!--[if lt IE 9]>
    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
    <title>Gene Dataset Correlation by jdittmar</title>
  </head>

  <body>
    <div id="container">
      <div class="inner">

        <header>
          <h1>Gene Dataset Correlation</h1>
          <h2>Find significant correlations between your gene of interest and other genes in the human genome</h2>
        </header>

        <section id="downloads" class="clearfix">
          <a href="https://github.com/jdittmar/CCLE---COLT-correlation/zipball/master" id="download-zip" class="button"><span>Download .zip</span></a>
          <a href="https://github.com/jdittmar/CCLE---COLT-correlation/tarball/master" id="download-tar-gz" class="button"><span>Download .tar.gz</span></a>
          <a href="https://github.com/jdittmar/CCLE---COLT-correlation" id="view-on-github" class="button"><span>View on GitHub</span></a>
        </section>

        <hr>

        <section id="main_content">
          <h1>
<a name="ccle---colt-correlation" class="anchor" href="#ccle---colt-correlation"><span class="octicon octicon-link"></span></a>CCLE---COLT-correlation</h1>

<p>The purpose of this program is to calculate the correlation between
CCLE gene expression data and COLT RNAi knockdown scores across multiple cell lines.
In processData a single gene's correlation is calculated againstevery other gene for which there is data. Just set the 'geneToAnalyze' the 'sourceData' (i.e. what data to use for the geneToAnalyze), and the 'compareDataTo'. SourceData and CompareDataTo can be either 'CCLE' or colt, but they cannot be the same. 'pValueCutoff' can be set to the correlation p-value threshold you would like to outputted results to be limited to 'tumorTypesToConsider' is a hash that can be used to specify the types of cell lines to consider (source tissue). If this parameter is omitted all tissues will be considered.</p>

<p><strong>Data sources:</strong></p>

<ul>
<li><p><strong>CCLE:</strong> <a href="http://www.broadinstitute.org/ccle/data/browseData?conversationPropagation=begin">http://www.broadinstitute.org/ccle/data/browseData?conversationPropagation=begin</a></p></li>
<li><p><strong>COLT:</strong> <a href="http://colt.ccbr.utoronto.ca/cancer/download.html">http://colt.ccbr.utoronto.ca/cancer/download.html</a></p></li>
</ul>
        </section>

        <footer>
          Gene Dataset Correlation is maintained by <a href="https://github.com/jdittmar">jdittmar</a><br>
          This page was generated by <a href="http://pages.github.com">GitHub Pages</a>. Tactile theme by <a href="https://twitter.com/jasonlong">Jason Long</a>.
        </footer>

        
      </div>
    </div>
  </body>
</html>